core/server/common/src/main/java/alluxio/master/journal/raft/JournalStateMachine.java

```diff
@@ -525,8 +525,8 @@ public synchronized long takeLocalSnapshot() {
           LOG.warn("Failed to rename snapshot from {} to {}", tempFile, snapshotFile);
           return RaftLog.INVALID_LOG_INDEX;
         }
-        LOG.info("Completed snapshot up to SN {} in {}ms", snapshotId,
-            System.currentTimeMillis() - lastSnapshotStartTime);
+        LOG.info("Completed snapshot with size {} up to SN {} in {}ms", snapshotFile.length(),
```

Comment:
this length is in bytes, so you can either:
1. convert to human readable format
2. or mention the size is in bytes
https://docs.oracle.com/javase/7/docs/api/java/io/File.html#length()